ajax/libs/angular-gridster/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "angular-gridster",
+  "description": "An implementation of gridster-like widgets for Angular JS. This directive gives you gridster behavior",
+  "filename": "angular-gridster.min.js",
+  "keywords": [
+    "angular",
+    "widget",
+    "gridster-like",
+    "angular directives"
+  ],
+  "author": "https://github.com/ManifestWebDesign/angular-gridster/graphs/contributors",
+  "homepage": "https://manifestwebdesign.github.io/angular-gridster",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ManifestWebDesign/angular-gridster.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ManifestWebDesign/angular-gridster.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
```

Comment:
1. Please modify 
```javascript
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      }
```
to
```javascript
      {
        "basePath": "dist",
        "files": [
          "**/*gridster*"
        ]
      }
```
to avoid fetching unnecessary files (.html, .txt , etc) in v0.9.2 ~ v0.9.8.
2. To fetch `src/angular-gridster.js` in v0.9.9 ~ v0.13.15 and `app/scripts/gridster.js`, `app/styles/gridster.css` respectively. Add
```javascript
      {
        "basePath": "src",
        "files": [
          "angular-gridster*.+(css|js|map)"
        ]
      },
      {
        "basePath": "app",
        "files": [
          "gridster*"
        ]
      }
```
following the first (above) change.
3. v.0.10.4 and v0.10.4 are totally the same, So we need to exclude one of them. Considering the correct tag scheme. We should exclude v.0.10.4. The way to exclude it is using [.gitignore ](https://github.com/cdnjs/cdnjs/blob/master/.gitignore). Please randomly choose one line with the comment `folders need to be ingored due to auto auto-update mechanism` and one line of end of this comment. Then add the line
```
/ajax/libs/angular-gridster/v.0.10.4
```

Thank you.

Memo: 
1. Fetch `src/angular-gridster.js` which is the non-minified version of `dist/angular-gridster.min.js` on the basis of https://github.com/ManifestWebDesign/angular-gridster/blob/master/Gruntfile.js#L90-L106
2. Only fetch `dist/*gridster*` and `app/+(styles|scripts)/gridster*`. Because we don't support .html, .txt, .htdaccess, .ico. and we have already hosted `jquery-ui.min.css`, all libraries composing scripts/modules.min.js(see https://github.com/ManifestWebDesign/angular-gridster/blob/v0.6.0/app/index.html#L43-L51) and es5-shim composing scripts/shims.min.js. As for `scripts/app.min.js` and `styles/main.css` are the instances of this requested library.

Reply:
@extend1994 "v.0.10.4 and v0.10.4 are totally the same" - are you talking about 0.10.3?

Reply:
@sashberd No I said v.0.10.4 and v0.10.4. The contents in these two versions are the same. But the author tag them twice.